NewPlatform.Flexberry.ORM.IntegratedTests/ICSSoft.STORMNET.Business.LINQProvider/LinqODataServiceTest.cs

```diff
@@ -119,29 +129,40 @@ public void TestODataIsOf()
         {
             foreach (IDataService dataService in DataServices)
             {
+                const string name = "n1";
                 ICSSoft.STORMNET.Windows.Forms.ExternalLangDef.LanguageDef.DataService = dataService;
                 var ds = (SQLDataService)dataService;
-                Plant2 cls1 = new Plant2() { Name = "n1" };
+                Plant2 cls1 = new Plant2() { Name = name };
                 Cabbage2 cls2 = new Cabbage2() { Name = "n2", Type = "type" };
 
                 var updateObjectsArray = new ICSSoft.STORMNET.DataObject[] { cls1, cls2 };
                 ds.UpdateObjects(ref updateObjectsArray);
-                var view = Plant2.Views.Plant2E;
-
-                IQueryable<Plant2> limit;
-                // limit = ds.Query<Plant2>(view).Where(it => it.__PrimaryKey != null);
-
-                LoadingCustomizationStruct lcs;
-                /*
-                lcs = LinqToLcs.GetLcs(limit.Expression, view);
-                lcs.LoadingTypes = new Type[] { typeof(Plant2), typeof(Cabbage2) };
-                lcs.LoadingTypes = new Type[] { typeof(Cabbage2) };
-                lcs.View = view;
-                var list = ds.LoadObjects(lcs);
-                */
-                // limit = ds.Query<Plant2>(view).Where(it => typeof(Cabbage2).IsInstanceOfType(it));
-                // lcs = LinqToLcs.GetLcs(limit.Expression, view);
-                // var list2 = ds.LoadObjects(lcs);
+                Assert.NotNull(updateObjectsArray);
+
+                View plantView = Plant2.Views.Plant2E;
+
+                IQueryable<Plant2> plant2s = ds.Query<Plant2>(plantView)
```

Comment:
I can not understand.

"IQueryable<Plant2> plant2s = ds.Query<Plant2>(plantView).Where(x => x.__PrimaryKey == cls1.__PrimaryKey);"

and

"Plant2 cl = new Plant2();
                cl.SetExistObjectPrimaryKey(cls1.__PrimaryKey);
                ds.LoadObject(plantView, cl);
                Assert.NotNull(cl);"

make the same. What for is there two same checks? Only one should be.